assets/extensions/examples/online-extensions.js

```diff
@@ -30,76 +32,73 @@ window.OPENSHIFT_CONSTANTS.HELP = {
   "default":                 "https://docs.openshift.com/online/latest/welcome/index.html"
 };
 
-/*
-  Custom angular module 
-*/
-angular.module('openshiftOnlineConsoleExtensions', ['openshiftConsole'])
-  .run(function(HawtioExtension) {
-
-    /*
-      Request system status from statuspage.io
-    */
-    var system_status_elem = $('<a href="http://status.openshift.com/" target="_blank" class="nav-item-iconic system-status project-action-btn" style="display: none;">');
-    var system_status_elem_mobile = $('<div row flex class="navbar-flex-btn system-status-mobile" style="display: none;">');
-
-    $.getJSON("https://m0sg3q4t415n.statuspage.io/api/v2/summary.json", function (data) {
-      var n = (data.incidents || [ ]).length;
-    
-      if (n > 0) {
-        var issueStr = n + ' open issue';
-        if (n !== 1) {
-          issueStr += "s";
+
+angular
+  // dont know if we really need openshiftOnlineConsoleExtensions,
+  // if the file exists, it will add the extensions, if it does not, nothing.
+  .module('openshiftConsole')
+  .run([
+    'extensionRegistry',
+    function(extensionRegistry) {
+
+      var system_status_elem = $('<a href="http://status.openshift.com/" target="_blank" class="nav-item-iconic system-status project-action-btn" style="display: none;">');
+      var system_status_elem_mobile = $('<div row flex class="navbar-flex-btn system-status-mobile" style="display: none;">');
+
+
+      $.getJSON("https://m0sg3q4t415n.statuspage.io/api/v2/summary.json", function (data) {
+        var n = (data.incidents || [ ]).length;
+        n = 32;
+        if (n > 0) {
```

Comment:
assuming you tested the issue badge appears correctly, if not just change this to n >= 0 temporarily and it will show up